.travis.yml

```diff
@@ -87,6 +87,7 @@ before_script:
 
 script:
   - ./run-tests.sh # E2E tests are skipped if E2E env is set to 'no'
+  - ./run-ext-tests.sh # test external services only
```

Comment:
Please use the same script with an option.

Reply:
done